src/Aspire.Dashboard/Components/Controls/ResourceSelect.razor.cs

```diff
@@ -0,0 +1,62 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Dashboard.Model;
+using Aspire.Dashboard.Model.Otlp;
+using Microsoft.AspNetCore.Components;
+using Microsoft.FluentUI.AspNetCore.Components;
+using Microsoft.JSInterop;
+
+namespace Aspire.Dashboard.Components.Controls;
+
+public partial class ResourceSelect
+{
+    private const int ResourceOptionPixelHeight = 32;
+    private const int MaxVisibleResourceOptions = 10;
```

Comment:
I played around with it and 10 felt too small. Trying a larger value.
```suggestion
    private const int MaxVisibleResourceOptions = 15;
```
btw thanks for going the extra mile to make this easily configurable.